c2corg_ui/static/js/outingediting.js

```diff
@@ -147,10 +147,6 @@ app.DocumentEditingController.prototype.formatOuting_ = function(outing) {
       outing.date_end = window.moment(outing.date_end).format('YYYY-MM-DD');
     }
 
-    var associations = outing.associations;
-    for (var i = 0; i < associations['users'].length; i++) {
-      associations['users'][i]['id'] = associations['users'][i]['document_id'];
```

Comment:
The API now uses ``document_id`` instead of ``id``for users as well. See https://github.com/c2corg/v6_api/commit/e0f61cde92e45830b856b3f930f4b3d5e1f7d271